packages/utils-demo/package.json

```diff
@@ -12,7 +12,7 @@
     "build-ts": "tsc -b tsconfig.build.json --verbose --pretty",
     "deploy-empty": "yarn build-ts && node dist/js/deployEmpty",
     "start": "yarn build-ts && node dist/js/src/test",
-    "start-all": "concurrently --restart-tries 3 --restart-after 1500 -n eth,backend,deploy \"yarn start-ganache\" \"yarn start-test-backend\" \"yarn start\"",
+    "start-all": "concurrently --restart-tries 3 --restart-after 1500 -n eth,backend,deploy \"yarn start-ganache\" \"yarn start-test-backend\" \"sleep 10 && yarn start\"",
```

Comment:
Use `\"wait-on tcp:8545 && yarn start\"` instead of `sleep`

Reply:
Thanks. These are some leftovers from my testing, not related to this PR (this PR is still WIP).